packages/frontend/editor-ui/src/components/ApiKeyCreateOrEditModal.vue

```diff
@@ -119,7 +119,9 @@ onMounted(() => {
 		const apiKey = apiKeysById[props.activeId];
 		label.value = apiKey.label ?? '';
 		apiKeyCreationDate.value = getApiKeyCreationTime(apiKey);
-		selectedScopes.value = !apiKeyScopesEnabled.value ? apiKeyStore.availableScopes : apiKey.scopes;
+		selectedScopes.value = !apiKeyScopesEnabled.value
+			? apiKeyStore.availableScopes
+			: apiKey.scopes.filter((scope) => apiKeyStore.availableScopes.includes(scope));
```

Comment:
Since we mistakenly added `execution:get` to all API keys in the initial migration, filter out all the scopes that are not in available list. Without this we would have to do another migration, which seems an unnecessary risk as this  filter should have been in place anyways.